cmds/core/netstat/netstat_linux.go

```diff
@@ -261,71 +261,71 @@ func evalProtocols(tcp, udp, udpl, raw, unix, ipv4, ipv6 bool) ([]netstat.Socket
 	if tcp && ipv4 {
 		t, err := netstat.NewSocket(netstat.PROT_TCP)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create TCP socket: %w", err)
```

Comment:
It might be more useful to say:
"evalProtocols creating TCP4 socket: %w"
because the error text will be what want wrong, and your text will be what you were trying to do.
Just this message alone, I would not be sure where to look.